sentry_sdk/tracing.py

```diff
@@ -472,8 +473,8 @@ def is_success(self):
         # type: () -> bool
         return self.status == "ok"
 
-    def finish(self, hub=None):
-        # type: (Optional[sentry_sdk.Hub]) -> Optional[str]
+    def finish(self, hub=None, **kwargs):
```

Comment:
just make it explicit `end_timestamp=None`, I don't like the opaqueness of kwargs and it's only one argument here anyway

Reply:
done.